desktop/cmp/dash/canvas/DashCanvas.js

```diff
@@ -6,17 +6,17 @@ import PT from 'prop-types';
 
 import ReactGridLayout, {WidthProvider} from 'react-grid-layout';
 import 'react-grid-layout/css/styles.css';
-import './DashReport.scss';
-import {DashReportModel} from './DashReportModel';
-import {dashReportContextMenu} from './impl/DashReportContextMenu';
-import {dashReportView} from './impl/DashReportView';
+import './DashCanvas.scss';
+import {DashCanvasModel} from './DashCanvasModel';
+import {dashCanvasContextMenu} from './impl/DashCanvasContextMenu';
+import {dashCanvasView} from './impl/DashCanvasView';
 
 const reactGridLayout = elemFactory(WidthProvider(ReactGridLayout));
 
-export const [DashReport, dashReport] = hoistCmp.withFactory({
-    displayName: 'DashReport',
-    className: 'xh-dash-report',
-    model: uses(DashReportModel),
+export const [DashCanvas, dashCanvas] = hoistCmp.withFactory({
+    displayName: 'DashCanvas',
+    className: 'xh-dash-Canvas',
```

Comment:
Should be lowercase

Reply:
Thanks for catching these-- casualty of find / replace